scripts/SubsetSelection/src/encoders/bge.py

```
@@ -0,0 +1,276 @@
+    
+    # Test each model variant
+    for model_name in MODEL_CONFIGS.keys():
+        print(f"\nTesting with model: {model_name}")
+        
+        try:
+            # Initialize encoder
+            encoder = UnifiedBGEEncoder(
+                model_name=model_name,
+                batch_size=2,
+                use_default_instruction=True  # Use model's default instruction
+            )
```

Pylint (3.3.8)

[error] Unexpected keyword argument 'batch_size' in constructor call

(E1123)